packages/designmanual/stories/simple-components.jsx

```diff
@@ -620,12 +620,41 @@ storiesOf('Enkle komponenter', module)
         </p>
       </StoryIntro>
       <StoryBody>
+        <h3>Logo uten url</h3>
         <Logo
           cssModifier="large"
           name
-          to="#"
           altText="Nasjonal digital læringsarena"
         />
+        <code style={{ marginLeft: 0 }}>
```

Comment:
Synes det holder å kun vise logo'en her (slik det var tidligere). Muligheten for å legge inn en link er en implementasjondetalj som vi stort sett ikke har dokumentert i andre eksempler.